source/auth/auth.rst

```diff
@@ -763,23 +764,29 @@ database command error.)
 Step 4
 ------
 
-To test SASLprep behavior, create a user with username and password equal to
-the Unicode character ROMAN NUMERAL NINE.  To create the user, use the
-(post-SASLprep) username and password "IX" and specify SCRAM-SHA-256
-credentials:
+To test SASLprep behavior, create two users:
+
+#. username: "IX", password "IX"
+#. username: "\u2168" (ROMAN NUMERAL NINE), password "\u2163" (ROMAN NUMERAL FOUR)
```

Comment:
Now that we're prohibiting SASLPrep for usernames, let's choose a non-ASCII username that is *not* equivalent to the ASCII one.

Reply:
It's intentional.  If an implementation does SASLprep when it shouldn't, "\u2168" will map to "IX" but have the wrong password so login will fail.

Reply:
Oh, the password is different. I missed that NINE is not FOUR. LGTM.